admin/testbrew.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Script executed by Buildbot on the Mac mini build slave, this instantiates the
+OS X Virtual Machine to a snapshot with a clean homebrew install, then executes
+via SSH the brew update, install and test commands to verify a successful
+installation of a homebrew recipe.
+
+The host machine must have:
+        * A VMWare OS X VM available at a particular location
+
+The guest virtual machine must have:
+        * A snapshot with a particular name, homebrew-clean
+        * Homebrew installed and available
+"""
+
+import os
+import sys
+import urllib2
+
+from twisted.python.usage import Options, UsageError
+
+from subprocess import check_output, CalledProcessError
```

Comment:
nit pick - it's nice to have all stdlib imports grouped together, followed by all third-party imports grouped together, followed by all internal imports grouped together.  :question: